guide/payments/transactions.md

```diff
@@ -0,0 +1,95 @@
+
+## Transaction lifecycle
+
+Let's lay out the entire payment process.
+
+{% include picture.html
+   image = "/assets/images/guide/payments/transactions/tx-lifecycle-simplified.svg"
+   mobile = "/assets/images/guide/payments/transactions/tx-lifecycle-simplified-mobile.svg"
+   alt-text = "Flow chart of the process of a transaction getting created and then finally confirmed"
+   width = 1350
+   height = 242
+   layout = "full-width"
+%}
+
+#### 1. Get recipient address
+The sender needs a valid address for the payment to be sent to. This can be shared by the recipient as a QR code, in plain text or as a payment link.
+
+#### 2. Creation
+The wallet application guides the sender through collecting the required information (address and amount), and any optional configurations (which coins to send, fee options) in order to create a transaction.
+
+#### 3. Signing
+The transaction needs to be signed by the [private key(s)]({{ '/guide/glossary/#private-key' | relative_url }}) of the input [address(es)]({{ '/guide/glossary/#address' | relative_url }}) to be valid. The signing is often done in the same application after the transaction has created and configured, but this does not have to be the case.
+
+#### 4. Broadcasting
+The transaction is broadcasted to a Bitcoin node, normally the one the wallet is connected to.
+
+#### 5. Validation
+The receiving node checks that the transaction is valid. In practice this means confirming that it was signed by the private key of the relevant address.
+
+#### 6. Propagation
+Once validated, the node passes the transaction on to other nodes in the network. The transaction is now in the *memory pool*, and remains there until it is mined. At this stage the transaction has 0 confirmations.
+<!-- indicate that a tx with a fee rate that is lower than the current normal can be stuck and even forgotten by miners if it remains in the mempool for too long -->
+
+#### 7. Confirmations
+Given that you know [how transactions are confirmed]({{ '/guide/getting-started/technology-primer/#how-are-transactions-confirmed' | relative_url }}), lets look at how the number of confirmations affects the payment settlement.
+
+Not every miner creates the new block with the same transactions, so some nodes may have a different version of the blockchain than others for a short time. The Bitcoin protocol's main function is to bring all nodes to the same version of the blockchain. Through a process called chain reorganization, nodes remove their incorrect block and update with the winning block as determined by the majority of other nodes.
+
+There is a slight risk that a transaction with 1 confirmation may revert to 0 confirmaations when a chain reorganisation occurs. Due to this, some parties may require more confirmations for the transaction before providing the product or service.
```

Comment:
Standardization to 🇺🇸 - 🇬🇧 English for consistency.
```suggestion
There is a slight risk that a transaction with 1 confirmation may revert to 0 confirmations when a chain reorganization occurs. Due to this, some parties may require more confirmations for the transaction before providing the product or service.
```

Reply:
damn ye sorry force of habit and growing up with british spelling, i need to setup my text editor.